lib/galaxy/tools/__init__.py

```diff
@@ -808,6 +808,7 @@ def parse(self, tool_source, guid=None, dynamic=False):
         self.containers = containers
 
         self.citations = self._parse_citations(tool_source)
+        self.xrefs = self._parse_xrefs(tool_source)
```

Comment:
Shouldn't this be:
```suggestion
        self.xrefs = tool_source.parse_xrefs()
```
? Otherwise the methods in lib/galaxy/tool_util/parser/xml.py and lib/galaxy/tool_util/parser/yaml.py are unused.